multi-repo-action/org-workflow-add.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// *****SET THESE PARAMETERS*****
+const ORG_NAME string = "organization name"
+const PAT string = "access token"
```

Comment:
his PAT will only need read permission, is this correct?

I wonder if there's a way to do this in a workflow, to avoid having users install golang, etc. My question I think is: can the default `secrets.GITHUB_TOKEN` be used? This way, the org maintainers need not create a PAT, need to have to revoke it, etc.

Just something to keep in mind, not this PR :-)

Reply:
Yes, only required permissions are public_repo and I think also read:org?